Explorer/Assets/DCL/Landscape/Worlds/WorldTerrainGenerator.cs

```diff
@@ -61,150 +41,75 @@ public void Dispose()
 
         public bool Contains(Vector2Int parcel)
         {
-            if (IsInitialized)
-                return terrainModel.IsInsideBounds(parcel);
+            if (TerrainModel != null)
+                return TerrainModel.IsInsideBounds(parcel);
 
             return false;
         }
 
         public float GetHeight(float x, float z) =>
```

Comment:
Please check both implementations as before they were different btu probably now they are not so classes could be unified under a single implementation. The same applies to `Contains` method as well